app/controllers/meeting_types_controller.rb

```diff
@@ -0,0 +1,40 @@
+class MeetingTypesController < ApplicationController
+  api :GET, '/meeting_types', 'This is the index view for all meeting types'
+  def index
+    meeting_types = MeetingType.all
+    render json: meeting_types, adapter: :json_api, status: :ok
+  end
+
+  api :GET, '/meeting_types', 'Gets the show page for a meeting type'
+  def show
+    meeting_type = MeetingType.find(params[:id])
```

Comment:
- [ ] I suggest that you use an instance variable for `@meeting_type` and set it in a private function called in a `before_action`, this way you can avoid doing this everywhere you need to. This is also a valid suggestion for the other controllers.